core/src/main/resources/error/error-classes.json

```diff
@@ -1,16 +1,50 @@
 {
+  "ALTER_TABLE_WITH_DROP_PARTITION_AND_PURGE_UNSUPPORTED_ERROR" : {
```

Comment:
Given that this is an error class, you can remove the `_ERROR` suffix to reduce redundancy and clutter.